docs/guides/http-client.md

```diff
@@ -2,19 +2,33 @@
 
 Prism includes a fully-featured HTTP Client that you can use to seamlessly perform requests to both a real server and a mocked document. The client is modeled after Axios so it may feel familiar.
 
-### Create from a filename or http resource
+### Create From Manual HTTP Operations
 
 ```ts
-const client = await createClientFromResource('examples/petstore.oas2.yaml', {
-  mock: true,
-  validateRequest: true,
-  validateResponse: true,
-});
+const { createClientFromOperations } = require('@stoplight/prism-http/dist/client');
+
+const client = createClientFromOperations(
+  [
+    {
+      method: 'get',
+      path: '/hello',
+      id: 'n1',
+      responses: [{ code: '200' }],
+    },
+  ],
+  { mock: true, validateRequest: true, validateResponse: true }
+);
 ```
 
-### Create from OpenAPI string
+To create the required operations array you can use two utilities function defined in the `@stoplight/prism-cli` package:
+
+### Create from file or HTTP resource
 
 ```ts
+const { default, getHttpOperationsFromResource } = require('@stoplight/prism-cli/dist/getHttpOperations')
```

Comment:
I thought about two alternatives:
1. `const { getHttpOperations, getHttpOperationsFromResource } = require('@stoplight/prism-cli/dist/operations')`
2. `const { getHttpOperations, getHttpOperationsFromResource } = require('@stoplight/prism-cli')` - why dig into internals at all?

Also, the default export might be misleading. I would name the `getHttpOperations` explicitly.